documentation/guide-batch-layer.asciidoc

```diff
@@ -212,8 +212,70 @@ Show help::
 As you can the each invocation includes the jdbc connection string to your database.
 This means that you have to make sure that the corresponding DB driver is in the classpath (the prepared jar only contains H2).
 
+== Authentication
+
+Most business application incorporate authentication and autorisation.
```

Comment:
`autorisation` should be `authorization` ;)

Reply:
yes!